Lib/copy.py

```diff
@@ -167,26 +170,12 @@ def deepcopy(x, memo=None, _nil=[]):
         _keep_alive(x, memo) # Make sure x lives at least as long as d
     return y
 
-_deepcopy_dispatch = d = {}
+_atomic_types =  {types.NoneType, types.EllipsisType, types.NotImplementedType,
```

Comment:
I don't think you need to remove the `d` variable. This is a cosmetic change and the reason `d` is introduced is to make the code shorter. The variable is also used for the copy part above, which would be slightly inconsistent if you change it.

Personally I don't think you should make unnecessary and unrelated changes to existing code. Sometimes it makes the code review harder, and it might confuse people when they check this commit in the future.

Reply:
I reverted the changes with the `d` variable.